test/smartwallet/smartWallet.test.ts

```diff
@@ -512,7 +681,7 @@ describe('SmartWallet', function(){
 
             expect(difference).approximately(amountToTransferAsNumber, 2);
         });
-    })
+    });
 
     describe.skip('Function recover()', function(){
         //TODO: This function is not implemented. Create test cases when it is.
```

Comment:
I know it was already there, but I'd remove this for now since it's not implemented